src/components/organization-members.tsx

```diff
@@ -264,7 +271,10 @@ export const OrganizationMembersTable = () => {
                     ))}
                 </Table.Tbody>
             </Table>
-            <Protect permission="org:sys_memberships:manage">
+            <Protect
+                permission="org:sys_memberships:manage"
+                fallback={<AccessDeniedAlert />}
+            >
```

Comment:
same comment as above, on these components we don't want to notify the user they lack permissions, instead we just don't render edit controls